guides/doc-Project_API/topics/con_overview-of-the-project-api.adoc

```diff
@@ -0,0 +1,18 @@
+[id="overview-of-the-{project-context}-api_{context}"]
+= Overview of the {Project} API
+
+The benefits of using the REST API are:
+
+* Broad client support {endash} any programming language, framework, or system with support for HTTP protocol can use the API.
+* Self-descriptive {endash} client applications require minimal knowledge of the {ProjectName} infrastructure because a user discovers many details at runtime.
+* Resource-based model {endash} the resource-based REST model provides a natural way to manage a virtualization platform.
+
+You can use the REST API to perform the following tasks:
+
+* Integrate with enterprise IT systems.
+* Integrate with third-party applications.
+* Perform automated maintenance or error checking tasks.
+* Automate repetitive tasks with scripts.
+
+As you prepare to upgrade {ProjectServer}, ensure that any scripts you use that contain {Project} API commands are up to date.
```

Comment:
unsure about the wording here. Maybe we better move this "hint" to the upgrade guide? aka. post-upgrade steps: ensure to update any scripts that use {Project} API"?

Reply:
Good idea.

Reply:
Adding to the upgrading guide in https://github.com/theforeman/foreman-documentation/pull/3245